libs/components/layout/src/lib/modules/back-to-top/back-to-top.component.spec.ts

```diff
@@ -205,7 +200,7 @@ describe('back to top component', () => {
 
     it('should scroll to target element when back to top button is clicked', () => {
       scrollElement(parentElement, 999, fixture);
-      const backToTopTarget = getBackToTopTarget();
+      const backToTopTarget = getBackToTopTarget()!;
```

Comment:
Nitpicky - but any way that we could change `isElementInView` to handle `null` so that we don't have to have these non-null assertions?

Reply:
updated